src/cargo/core/shell.rs

```diff
@@ -204,6 +225,16 @@ impl Shell {
         self.print(&status, Some(&message), Green, true)
     }
 
+    /// Shortcut to right-align and paint the query part of the output message as a green in stdout.
+    pub fn status_stdout_part_green(
+        &mut self,
+        name_no_query: Vec<&str>,
+        desc_no_query: Vec<&str>,
```

Comment:
Can you document what these arguments are? I can't figure out what these are from the signature/name of the method...

Reply:
Added explanation and changed parameter naming